.agents/skills/motion-design/reference/troubleshooting.md

```
@@ -0,0 +1,73 @@
+# Troubleshooting
+
+## Looks Robotic
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)